std/meta.d

```diff
@@ -1094,42 +1094,23 @@ template templateOr(Preds...)
     }
 }
 
+import std.traits : isIterable;
```

Comment:
Maybe the self-important template idiom could be used here to avoid the top level import.